scene/main/http_request.cpp

```diff
@@ -222,11 +222,10 @@ bool HTTPRequest::_handle_response(bool *ret_value) {
 				// New url, request all again
 				err = _parse_url(new_request);
 			} else {
+				err = _request();
```

Comment:
Not sure about this change, maybe the other branch does require both `_parse_url(new_request)` and the `_request` to be called. CC @Faless

Reply:
Done.